bindings/python/tools/python/ostk/astrodynamics/converters.py

```diff
@@ -128,3 +128,46 @@ def coerce_to_interval(
         start_instant=coerce_to_instant(value[0]),
         end_instant=coerce_to_instant(value[1]),
     )
+
+
+def coerce_to_datetime_tuple(
+    value: (
+        Interval
+        | tuple[Instant, Instant]
+        | tuple[datetime, datetime]
+        | tuple[str, str]
+        | str
+    ),
+) -> tuple[datetime, datetime]:
```

Comment:
I think this is more scalable in terms of function signature:

```suggestion
def coerce_to_datetime_tuple(
    value: (
        Interval
        | tuple[Instant | datetime | str , Instant | datetime | str]
        | str
    ),
) -> tuple[datetime, datetime]:
```

**Should be working without you having to change the implementation.**

Would just need to add a couple of "heterogeneous" test cases e.g. 

```python
        (
            (
                Instant.date_time(DateTime(2020, 1, 1), Scale.UTC),
                datetime(2020, 1, 2, tzinfo=timezone.utc),
            )
        ),
        (
            (
                datetime(2020, 1, 1, tzinfo=timezone.utc),
                "2020-01-02T00:00:00Z",
            )
        ),
```


I'd do the same with the coerce_to_interval signature